triage-party/release-team/deployment.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: triage-party
+  namespace: triageparty-release
+  labels:
+    app: triage-party
+spec:
+  selector:
+    matchLabels:
+      app: triage-party
+  template:
+    metadata:
+      labels:
+        app: triage-party
+    spec:
+      containers:
+        - name: triage-party
+          image: triageparty/triage-party:1.1.0
+          env:
+            - name: GITHUB_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  name: triage-party-github-token
```

Comment:
@spiffxp @justaugustus Can you add the token as a `secret` ?

Reply:
I got the token from @justaugustus, created a kubernetes secret, put that in secretmanager:
```
kubectl create secret generic triage-party-github-token --from-file token -o=yaml --dry-run \
  | gcloud secrets create --project kubernetes-public triage-party-github-token --replication-policy automatic --data-file -
```
Gave access to that secret to the same group that has access to this namespace:
```
gcloud secrets add-iam-policy-binding --project kubernetes-public triage-party-github-token --member group:k8s-infra-rbac-triageparty-release@kubernetes.io --role roles/secretmanager.admin
```

You can deploy this to the cluster via:
```
gcloud secrets versions access latest --project kubernetes-public --secret triage-party-github-token | kubectl apply -n triageparty-release -f -
```

Reply:
@spiffxp Unfortunately, members of this group don't have the privileges to create secrets : https://github.com/kubernetes/k8s.io/blob/f297d826dffe142def86d39a9077e707c97bebd1/infra/gcp/namespaces/namespace-user-role.yml#L41-L43.

Can you run the last command for us ?

Reply:
Done.  I also opened https://github.com/kubernetes/k8s.io/pull/988 to suggest that we give members of the group the ability to create/delete read secrets for that namespace.  We do that for pretty much all other namespace-scoped resources